app/v2/controllers/CustomJsonErrorHandler.scala

```diff
@@ -43,10 +45,11 @@ class CustomJsonErrorHandler @Inject() (
       super.onClientError(request, statusCode, message).map {
         _ =>
           message match {
-            case "ERROR_INVALID_GRN_COUNTRY_CODE" =>
+            case InvalidGRNCode =>
               BadRequest(Json.toJson(PresentationError.badRequestError("The guarantee reference number must be for a GB or XI guarantee.")))
-            case "ERROR_INVALID_GRN_FORMAT" =>
+            case InvalidGRNFormat =>
               BadRequest(Json.toJson(PresentationError.badRequestError("The guarantee reference number is not in the correct format.")))
+            case _ => BadRequest(Json.toJson(PresentationError.badRequestError(message)))
```

Comment:
This is going to return the unredacted message -- which is not what we want. We should just return whatever is coming into this map to keep it consistent with what HMRC does with other messages..